client/src/contexts/NewForm.jsx

```diff
@@ -1,25 +1,99 @@
-import React, { useState } from 'react';
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState, useContext, useEffect } from 'react';
+import axios from 'axios';
 import styled from 'styled-components';
 import PropTypes from 'prop-types';
+import Console from '../Console';
+import ReviewInput from '../components/Ratings/Review/ReviewInput';
 import { Modal, ModalParent, ModalClose } from './Shared.styled';
-// import Console from '../../Console';
+import { useMeta } from './ReviewMeta';
+import { ProductIDContext } from './ProductIDContext';
 
-// formtype: review, question
-export default function NewForm({ formtype, productName, showModal }) {
+export default function NewForm({
+  formtype, productName, showModal, setShowModal, ...args
+}) {
   let type;
-  const [modalStatus, setModalStatus] = useState(true);
-  // const [data, setData] = useState({});
-  // const handFormChange = () =
-  const review = {
+  let bodyNote;
+  const meta = useMeta();
+  const factorList = (meta) ? meta.characteristics : {};
+  // CONFIRM WITH ALEX this default won't imapct his section
+  const [data, setData] = useState({ recommendation: 'true' });
+  const productId = useContext(ProductIDContext);
```

Comment:
Will this need to change with the current product ID context?

Reply:
Just did in the lastest commit

Reply:
Haha, I just got the email notification